src/Microsoft.ML.Legacy/Runtime/EntryPoints/JsonUtils/GraphRunner.cs

```diff
@@ -23,14 +23,15 @@ public sealed class GraphRunner
         private readonly IHost _host;
         private readonly EntryPointGraph _graph;
 
-        public GraphRunner(IHostEnvironment env, ComponentCatalog catalog, JArray nodes)
+        public GraphRunner(IHostEnvironment env, JArray nodes)
         {
             Contracts.CheckValue(env, nameof(env));
+            var catalog = env.ComponentCatalog;
```

Comment:
>var catalog = env.ComponentCatalog; [](start = 12, length = 35)

This catalog variable only get validated, and I don't think this is necessary since it's part of contract.
Feel free to remove it with check.